test/jdk/java/net/Inet6Address/B6206527.java

```diff
@@ -40,22 +40,24 @@
     public static void main (String[] args) throws Exception {
         Inet6Address addr = getLocalAddr();
         if (addr == null) {
-            System.out.println ("Could not find a link-local address");
+            System.out.println("Could not find a link-local address");
             return;
         }
 
-        ServerSocket ss = new ServerSocket();
-        System.out.println ("trying LL addr: " + addr);
-        ss.bind(new InetSocketAddress(addr, 0));
+        try (ServerSocket ss = new ServerSocket()) {
```

Comment:
Maybe for brevity you could use `try (var ss = new ServerSocket()) {`

Only a suggestion. It's up to you.

Reply:
Though I agree that the brevity is nice, I think I will leave it as is (`ServerSocket` instead of `var`) in the interest of clarity on the type of instance being created in both test cases as it was the original version of the test.